### PR TITLE
fix: remove whilespace in hall-of-fame

### PR DESCRIPTION
**Pull request by animesh-sh**
My first open source contributions.
I just got rid of the extra whitespace, that way the document looks better.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **AWESOME! LGTM.** :sparkles: :sparkles: 

Thanks, @animesharma3, for this pull request and for contributing to the code-base for the first time.

We will be looking forward to more contributions in future.

Cheers and happy contributing!

File: HoF.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ We appreciate any responsible disclosure of vulnerabilities that might impact th
 
 - Mehul Mohan from [codedamn](https://codedamn.com) ([@mehulmpt](https://twitter.com/mehulmpt)) - [Vulnerability Fix](https://github.com/freeCodeCamp/freeCodeCamp/blob/bb5a9e815313f1f7c91338e171bfe5acb8f3e346/client/src/components/Flash/index.js)
 - Peter Samir https://www.linkedin.com/in/peter-samir/
-  > ### Thank you for your contributions :pray:
+> ### Thank you for your contributions :pray:
```
Comment (ShaunSHamilton) [inline HoF.md:7]:
```suggestion

> ### Thank you for your contributions :pray:
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for taking the time to open this. However, our Prettier config does not allow this format to pass our Lint checks - for good reason.

I suggest following [this section](https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=follow-these-steps-to-get-your-development-environment-ready) of the Contributor's Docs to see how to set up your local environment for contributing to this repository.

**Comment by raisedadead** [comment]
Hi, 

We are closing old issues/pull-requests that we think are stale. If you need still need help with this or are fixing this, please open a fresh issue/pull-request.

Thanks for your understanding. 

Happy contributing.

**Pull request closed without merging**